subsys/usb/device_next/usbd_endpoint.c

```diff
@@ -59,7 +59,20 @@ static void usbd_ep_ctrl_set_zlp(struct usbd_context *const uds_ctx,
 				 struct net_buf *const buf)
 {
 	struct usb_setup_packet *setup = usbd_get_setup_pkt(uds_ctx);
+	struct usb_device_descriptor *desc = uds_ctx->fs_desc;
 	size_t min_len = MIN(setup->wLength, buf->len);
+	uint8_t mps0 = 0;
```

Comment:
Initializing to 0 leads to taking remainder from division by zero later on. This would only trigger in the asserts-disabled case. Not sure if it would be better to just default to `USB_CONTROL_EP_MPS` instead of `0` here.